setup.py

```diff
@@ -74,6 +74,7 @@
 
 install_requires = [
     'arrow>=0.12.1',
+    'attrs>=17.4.0',
```

Comment:
What is this needed for?

Reply:
There is a compatibility issue between ``attrs`` and ``jsonschema`` with the lowest requirements in travis 
```python
class TypeChecker(object):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/jsonschema/_types.py", line 64, in TypeChecker
    _type_checkers = attr.ib(default=pmap(), converter=pmap)
TypeError: attrib() got an unexpected keyword argument 'converter'
```
```
jsonschema 3.0.1 has requirement attrs>=17.4.0, but you'll have attrs 17.3.0 which is incompatible.
```

https://travis-ci.org/inveniosoftware/invenio-records-rest/jobs/514094978
will investigate more